docs/change-log.md

```diff
@@ -2,6 +2,31 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update February 23rd, 2024
+
+- **New:** Introduced the [Delete Active Resources API](https://api-docs.humanitec.com/#tag/ActiveResource/operation/deleteActiveResource), allowing to detach an active resource. A detached resource remains intact but is no longer connected to the Platform Orchestrator.
+
+- **Fixed:** Switching frequently between Organizations no longer causes loading issues within an Application.
+
+#### Versioned component changes:
```

Comment:
Could we add please?
- `humctl` 0.13.5 - https://github.com/humanitec/cli/releases/tag/v0.13.5
- `score-compose` 0.9.0 -https://github.com/score-spec/score-compose/releases/tag/0.9.0